R/daemons.R

```diff
@@ -40,8 +40,8 @@
 #'     \code{\link{ssh_config}}.
 #' @param dispatcher [default TRUE] logical value whether to use dispatcher.
 #'     Dispatcher is a local background process that connects to daemons on
-#'     behalf of the host and ensures FIFO scheduling (see Dispatcher section
-#'     below).
+#'     behalf of the host and ensures FIFO scheduling. Specify NA for threaded
```

Comment:
FWIW I think you're running in to https://design.tidyverse.org/boolean-strategies.html

Reply:
You're right of course. I'm simply using NA here as there _is_ a third logical option. Will think on it more as the intention is to introduce it now as experimental, but in time it would likely become and replace the default TRUE option.

Reply:
Thanks @hadley. I'm about to merge a PR which changes the argument to `dispatcher = c("process", "thread", "none")`. This will really ease future migration plans, as having `dispatcher = NA` be default would be awkward to say the least.